docs/classes.rst

```diff
@@ -482,7 +482,8 @@ The binding code for this example looks as follows:
         .value("Cat", Pet::Kind::Cat)
         .export_values();
 
-    py::class_<Pet::Attributes> attributes(pet, "Attributes")
+    py::class_<Pet::Attributes> attributes(pet, "Attributes");
```

Comment:
Thanks for reporting! This is clearly a bug in the docs, but I think it would be better to delete ` attributes` because it's not used:
```
py::class_<Pet::Attributes>(pet, "Attributes")
```

Could you please update this PR to make that change?